.github/workflows/continuous-delivery.yml

```diff
@@ -3,8 +3,10 @@ name: continuous-delivery
 on:
   push:
     branches:
-      - '**'
+      - main
+      - 'dependabot/**'
   pull_request:
+    types: [ opened ]
```

Comment:
i think this should be `action`, not `types` right?
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request

Reply:
I can answer my own question:
From the GH docs.
>By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. To trigger workflows by different activity types, use the types keyword.